.gitignore

```diff
@@ -3,3 +3,13 @@
 *.zip
 
 !.vscode/settings.json
+
+# Sealed Secrets
+sealed-secrets-master-key-backup.yaml
+sealed-secrets-backup-*.yaml
+public-key.pem
+temp-private-key.yaml
+
+# Plain text secrets (use .example.yaml templates instead)
+home-assistant/secrets.yaml
```

Comment:
I meant move to .gitignore inside the appropriate directory instead of global file.